robottelo/hosts.py

```diff
@@ -61,6 +61,20 @@ def get_sat_version():
     return Version('9999' if 'nightly' in sat_version else sat_version)
 
 
+def get_sat_rhel_version():
+    """Try to read rhel_version from Satellite host
+    if not available fallback to DynaConfs."""
```

Comment:
We might decide to change the configuration manager in the future. So I would keep the names general.
```suggestion
    if not available fallback to robottelo configuration."""
```

Reply:
Updated as suggested.